examples/music-v2.py

```diff
@@ -33,11 +33,12 @@ def __unload(self):
 
         # Clears the players from Lavalink's internal cache
         self.bot.lavalink.players.clear()
-        self.bot.lavalink.unregister_hook(self._track_hook)

```

Comment:
I think it's better to also call `self.bot.lavalink.destroy()` to safely disconnect websockets.